apps/distgo/config/config.go

```diff
@@ -256,11 +256,23 @@ type DockerDep struct {
 	TargetFile string `yaml:"target-file" json:"target-file"`
 }
 
+type DockerImageInfo struct {
```

Comment:
Would move under declaration of `DockerImage` (and next to `SLSImageInfo`) for consistency